### ISSUE TITLE
Too little space on flash

### ISSUE THREAD

**Issue by drasko** [nodesign/weio#241]
I have noticed that we have critically low flash space after v1.1.

Quick look:
```
root@WEIO:/weio/www/libs# du -h .
51.5K   ./ace/snippets
1.4M    ./ace
267.0K  ./bootstrap/css
21.5K   ./bootstrap/img
89.0K   ./bootstrap/js
377.5K  ./bootstrap
111.5K  ./bootstrap-3.3.0/css
35.0K   ./bootstrap-3.3.0/js
146.5K  ./bootstrap-3.3.0
20.0K   ./chartJS
27.0K   ./css
25.5K   ./flatdoc/theme-white
79.5K   ./flatdoc
126.5K  ./font-awesome/css
411.5K  ./font-awesome/font
0       ./font-awesome/less
59.0K   ./font-awesome/scss
597.0K  ./font-awesome
90.5K   ./jqtree
443.5K  ./jquery
33.5K   ./sockJS
1.0K    ./weio/boilerPlate
142.0K  ./weio
3.4M    .
```

Can we reduce `ace`, `font-awesome` and `jquery` size?

**Comment by drasko** [comment]
BTW, right after update:
```
root@WEIO:/# df
Filesystem           1K-blocks      Used Available Use% Mounted on
rootfs                    2816      1280      1536  45% /
/dev/root                12288     12288         0 100% /rom
tmpfs                    30948       256     30692   1% /tmp
tmpfs                    30948       924     30024   3% /tmp/root
tmpfs                      512        36       476   7% /dev
/dev/mtdblock5            2816      1280      1536  45% /overlay
overlayfs:/overlay        2816      1280      1536  45% /
/dev/sda1              3877888     13128   3864760   0% /weioUser/sd
```

**Comment by drasko** [comment]
I have run also to this problem: https://forum.openwrt.org/viewtopic.php?id=39016

**Comment by drasko** [comment]
This will be resolved with new version of OpenWrt. WeIO will use official Chaos Calmer and will offload package management to `opkg` instead havim them hard-compiled into the image.

v1.2 is currently in preparation and should be released soon.

**Comment by ks156** [comment]
extroot fix this problem : https://github.com/nodesign/weio/wiki/How-to-extend-WeIO-flash-space

**Comment by ks156** [comment]
Fixed by v1.2
